src/backend/marsha/core/management/commands/clean_medialive_dev_stack.py

```diff
@@ -0,0 +1,73 @@
+"""Clean medialive stack for developement environment."""
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from marsha.core.utils.medialive_utils import (
+    delete_mediapackage_channel,
+    list_medialive_channels,
+    medialive_client,
+    mediapackage_client,
+)
+from marsha.core.utils.time_utils import to_datetime
+
+
+class Command(BaseCommand):
+    """Clean medialive stack for development environment."""
+
+    help = (
+        "Check every resources created on medialive and mediapackage and remove all resources"
+        "belonging to a development environment created more than x minutes ago."
+    )
+
+    def handle(self, *args, **options):
+        """Execute management command."""
+        now = timezone.now()
+
+        for medialive_channel in list_medialive_channels():
+            tags = medialive_channel.get("Tags", {})
+            # the channel must be a marsha one and the environment must start with dev-
+            if tags.get("app") == "marsha" and tags.get("environment", "").startswith(
+                "dev-"
+            ):
+                # the channel name contains the environment, the primary key and the created_at
+                # stamp. Here we want to use the timestamp
+                environment, pk, stamp = medialive_channel["Name"].split("_")
+                created_at = to_datetime(stamp)
+
+                if (
+                    created_at + timedelta(seconds=settings.NB_SECONDS_LIVING_DEV_STACK)
+                    <= now
+                ):
+                    self.stdout.write(
+                        f"Cleaning stack with name {medialive_channel['Name']}"
+                    )
+
+                    if medialive_channel["State"] == "RUNNING":
```

Comment:
Use the state constant?

Reply:
The constant is in low case. we change the case in the `medialive` lambda function.

Reply:
ow ok. Is there a good reason why we have to change the case?

Reply:
To be honest I don't remember. To be consistent with the other existing state ?